internal/gossiper/proposer.go

```diff
@@ -193,7 +193,7 @@ func (g *Proposer) HandleAppGossip(ctx context.Context, nodeID ids.NodeID, msg [
 	var seen int
 	for _, tx := range txs {
 		// Verify signature async
-		txDigest, err := tx.Digest()
+		txDigest, err := tx.UnsignedBytes()
```

Comment:
Can we rename `txDigest` to `unsignedTxBytes` ?

Reply:
done